docs/data/material/components/buttons/buttons.md

```diff
@@ -10,141 +10,147 @@ unstyled: /base-ui/react-button/
 
 # Button
 
-<p class="description">Buttons allow users to take actions, and make choices, with a single tap.</p>
-
-Buttons communicate actions that users can take. They are typically placed throughout your UI, in places like:
-
-- Modal windows
-- Forms
-- Cards
-- Toolbars
+<p class="description">Buttons enable users to take actions and make choices.</p>
 
 {{"component": "modules/components/ComponentLinkHeader.js"}}
 
-## Basic button
-
-The `Button` comes with three variants: text (default), contained, and outlined.
-
-{{"demo": "BasicButtons.js"}}
+## Introduction
 
-### Text button
+Buttons communicate actions that users can take.
+The Material UI Button component replaces the native HTML `<button>` element and offers expanded options for styling and accessibility.
 
-[Text buttons](https://m2.material.io/components/buttons#text-button)
-are typically used for less-pronounced actions, including those located: in dialogs, in cards.
-In cards, text buttons help maintain an emphasis on card content.
+{{"demo": "IntroButtons.js"}}
 
-{{"demo": "TextButtons.js"}}
+## Basics
```

Comment:
```suggestion
## Basics

### Import
```

Should we add it? We did it on the Snackbar page, but I guess we never formalized it as part of the template!

Reply:
Uhm, that makes sense, yeah! Well, I have no strong preferences either, so I'm happy to go without it, too.

Reply:
Yeah, these first two Base UI results, for example, are from the Components API and Hooks API tabs, respectively, which makes me think we should probably include them somehow in the results (so you know what that's about).

Reply:
The problem here to me is how we rank Algolia results. If I'm on a Material UI page, no way I want to see Base UI or MUI System results in between. I want first as section that is all about the product I'm looking at, and second an area for other results.

Reply:
I think that's _another_ problem on top of the one Sam shared. Imagine if Sam's screenshot is a search result when browsing through the Base UI docs. The results order is correct, but still, there's an opportunity to improve how we differentiate pages or sections with the same heading text.